pages/jobs-form.tsx

```diff
@@ -1,12 +1,15 @@
 import Layout from "../components/Layout";
 import FormField from "../components/FormField";
 
+const pageDescription =
+  "Add a job opportunity and have it featured on the Jobs Board to get in touch with local talent.";
+
 export default function JobsForm() {
   const BASE_URL = process.env.NEXT_PUBLIC_VERCEL_URL;
   return (
-    <Layout page="Submit a Job">
+    <Layout page="Submit a Job" description={pageDescription}>
       <h1 className="text-3xl p-2">
-        Add a listing to be featured on the Jobs Board
+        Add a listing to be featured on the Code Youngstown jobs board
```

Comment:
The H1 can probably be just "Submit a Job".

We can add something like "Submit a job to be featured on the Code Youngstown Job Board" as an H2